docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -34,6 +34,28 @@ The json payload for create and update operations cannot contain keys beginning
 
 @@@
 
+## Nexus metadata
+
+When using the endpoints described on this page, the responses can contain the following metadata:
+
+- `_self`: unique address of this resource in the current Nexus Delta deployment
```

Comment:
"current deployment" is probably not helpful, won't most users not understand what this means?

Reply:
Simplified